### DISCUSSION TITLE
Is it possible to generate v3.1.0 OpenAPI/Swagger docs from a Supabase project?

### DISCUSSION THREAD

**Discussion by drewbietron** [supabase/supabase#18863] [Questions]
Have been messing with adding a Supabase project's API as an action for OpenAI's new GPT Builder.  The GPT Builder accepts Swagger docs in order to add an action that will query the provided API. 

It appears that it only accepts version 3.1.0 Swagger docs at the moment but I cannot find a way to export anything other than the 2.0 docs available from https://supabase_project.com/rest/v1/?apikey=key

Any help would be appreciated!

**Comment by fluxsaas** [comment]
try this:

Paste your OpenAPI 2.0 definition into https://editor.swagger.io/ and select Edit > Convert to OpenAPI 3 from the menu.

![Screenshot 2024-01-11 at 12 16 18](https://github.com/supabase/supabase/assets/13223/59a0c262-1dda-4ad6-b67b-40fb59f95785)


see https://stackoverflow.com/questions/59749513/how-to-convert-openapi-2-0-to-openapi-3-0
